pkgs/development/compilers/graalvm/community-edition/default.nix

```diff
@@ -12,9 +12,11 @@ let
   /*
     Looks a bit ugly but makes version update in the update script using sed
     much easier
+
+    Don't change these values! They will be updated by the update script, see ./update.nix.
   */
-  graalvm11-ce-release-version = "22.0.0.2";
-  graalvm17-ce-release-version = "22.0.0.2";
+  graalvm11-ce-release-version = "22.1.0";
+  graalvm17-ce-release-version = "22.1.0";
   graalvm11-ce-dev-version = "22.2.0-dev-20220415_1945";
   graalvm17-ce-dev-version = "22.2.0-dev-20220415_1945";
```

Comment:
You can delete the `graalvmXX-ce-dev-version` likes I think.

Reply:
Thanks :)